index.js

```diff
@@ -14,7 +22,18 @@ function hide(args) {
 // show elements
 function show(args) {
   args.forEach((arg) => {
-    arg.style.display = 'block';
+    arg.style.display = 'flex';
+  });
+}
+
+// blur elements
+function blurElements(args, state) {
```

Comment:
- [ ] OPTIONAL
It will be a nice idea to prefer `arrow function` over the normal JS `function`, so as to make it simpler and cleaner synyax